input/k8s/k8s_test.go

```diff
@@ -135,7 +138,7 @@ func TestPublishK8sNodeAsset(t *testing.T) {
 				internal.WithAssetKindAndID(tt.assetKind, tt.assetID),
 				internal.WithAssetType(tt.assetType),
 				internal.WithAssetParents(tt.parents),
-				internal.WithNodeData(tt.assetName, &startTime),
+				internal.WithNodeData(tt.assetName, tt.assetStatusReady, &startTime),
```

Comment:
In all the 3 CSPs, we report the status of cloud instances under `asset.metadata.state` ([example](https://github.com/elastic/assetbeat/blob/main/input/aws/ec2.go#L91)). 

I do not have strong opinions regarding where the node status should be reported. However, I wonder if it would be beneficial to have consistency on where the "state" of an asset is published, when applicable. What do you think?

Reply:
@girodav The main reason I used the `kubernetes.node.status.ready` is that it is also part of the signals. And could help to keep the latest value between documents collected by assetbeat and implicit collector.
The `asset.metadata.state` may make more sense as a field name (and requires different logic to feel its value) but will only be part of assetbeat assets. 
Maybe we could introduce the same logic for `asset.metadata.state` in implicit collector as well. 
Wdyt?